root/static/scripts/edit/URLCleanup.js

```diff
@@ -1317,6 +1317,19 @@ const CLEANUPS = {
       return {result: false};
     },
   },
+  'dnb': {
+    match: [
+      new RegExp('^(https?://)?([^/]+\\.)?d-nb\\.info', 'i'),
```

Comment:
I don't understand, it seems that we need this line to retrict link type to artists. Could you explain further?

Reply:
There's a separate rule in the same file, just named otherdatabases, that also matches d-nb.info. Since you're moving this to be its own rule, it shouldn't also be matched there :)